torch_xla/distributed/spmd/xla_sharding.py

```diff
@@ -508,29 +505,19 @@ def mark_sharding(t: Union[torch.Tensor, XLAShardedTensor],
   assert len(t.shape) == len(partition_spec), \
     f"Partition spec length ({len(partition_spec)}) should be equal to the input rank ({len(t.shape)})."
 
+  op_sharding = mesh.get_op_sharding(partition_spec)
+  tile_assignment, group_assignment, replication_groups, sharding_type = _extract_op_sharding_specs(
```

Comment:
These op sharding specs are only needed for dynamo custom op, so maybe we can move this extraction under the if statement below?

Reply:
Good catch, done